src/fsm.c

```diff
@@ -4104,7 +4108,7 @@ __ni_ifworker_refresh_modems(ni_fsm_t *fsm)
 	}
 
 	for (object = list_object->children; object; object = object->next) {
-		ni_fsm_recv_new_modem(fsm, object, FALSE);
+		ni_fsm_recv_new_modem(fsm, object, TRUE);
```

Comment:
From the code "pattern" I'm missing here https://github.com/openSUSE/wicked/blob/468a60bcf107b7375702490a4a1171860429d852/src/fsm.c#L4125 a 
```
if (modem) ni_modem_reset(modem);
```
WDYT?

Reply:
Did you noticed the `#ifdef MODEM` around these functions that disables incomplete modem functions/support completely?

Reply:
ok, where do I find the the `define`'s?

Reply:
e.g. in https://github.com/openSUSE/wicked/blob/468a60bcf107b7375702490a4a1171860429d852/src/fsm.c#L4092
There is no `AC_ARG_ENABLE([modem], ...)` + `AC_DEFINE([MODEM], ...` in  configure.ac that would permit to enable it -> dead code.

Reply:
thx for explanation!